src/fork_gc.c

```diff
@@ -728,6 +729,14 @@ void ForkGc_ReadGarbageFromFork(ForkGCCtx *gc, int *ret_val) {
   RedisModule_FreeThreadSafeContext(rctx);
 }
 
+static int ForkGc_IsForkApiExists() {
+  return false;
+}
+
+static int ForkGc_Fork() {
```

Comment:
I'm not sure why we need a wrapper; doesn't every POSIX system support `fork()`?

Reply:
We will soon have a redis module fork api (the redis will do the fork for us) I made it so we will have it ready for this.